JSONLD/organisations/consortia/solaris-heppa.json

```diff
@@ -0,0 +1,374 @@
+                    "to": 2022,
+                    "phase": "CMIP6"
+                },
+                {
+                    "from": 2025,
+                    "to": "-",
+                    "phase": "CMIP7"
+                }
+            ]
+        },
+        {
+            "@type": "consortia:member",
+            "consortia:institution": {
+                "@id": "mip-cmor-tables:organisations/institutions/lasp"
+            },
+            "consortia:membership_type": "current",
+            "consortia:dates": [
+                {
+                    "from": 2016,
+                    "to": 2022,
+                    "phase": "CMIP6"
+                },
+                {
+                    "from": 2025,
+                    "to": "-",
+                    "phase": "CMIP7"
+                }
+            ]
+        },
+        {
+            "@type": "consortia:member",
+            "consortia:institution": {
+                "@id": "mip-cmor-tables:organisations/institutions/lpc2e"
+            },
+            "consortia:membership_type": "current",
+            "consortia:dates": [
+                {
+                    "from": 2016,
+                    "to": 2022,
+                    "phase": "CMIP6"
+                },
+                {
+                    "from": 2025,
+                    "to": "-",
+                    "phase": "CMIP7"
+                }
+            ]
+        },
+        {
+            "@type": "consortia:member",
+            "consortia:institution": {
+                "@id": "mip-cmor-tables:organisations/institutions/metoffice"
+            },
+            "consortia:membership_type": "current",
+            "consortia:dates": [
+                {
+                    "from": 2016,
+                    "to": 2022,
+                    "phase": "CMIP6"
+                }
+            ]
+        },
+        {
+            "@type": "consortia:member",
+            "consortia:institution": {
+                "@id": "mip-cmor-tables:organisations/institutions/mps"
+            },
+            "consortia:membership_type": "current",
+            "consortia:dates": [
+                {
+                    "from": 2016,
+                    "to": 2022,
+                    "phase": "CMIP6"
+                },
+                {
+                    "from": 2025,
+                    "to": "-",
+                    "phase": "CMIP7"
+                }
+            ]
+        },
+        {
+            "@type": "consortia:member",
+            "consortia:institution": {
+                "@id": "mip-cmor-tables:organisations/institutions/ncar"
+            },
+            "consortia:membership_type": "current",
+            "consortia:dates": [
+                {
+                    "from": 2016,
+                    "to": 2022,
+                    "phase": "CMIP6"
+                },
+                {
+                    "from": 2025,
+                    "to": "-",
+                    "phase": "CMIP7"
```

Comment:
NCAR is a contributor to AMIP1, AMIP2, CMIP1, CMIP2, CMIP2+, CMIP3, [CMIP4 which has never been formally identified], CMIP5, CMIP6, and plans to contribute to CMIP7, probably CMIP8, ... just to consider how we'd use this field